### ISSUE TITLE
Show Loading Status

### ISSUE THREAD

**Issue by imshvc** [jurplel/qView#715]
When opening big files, the window stays empty until the entire file has loaded into memory.
This causes confusion:
  1. The file is not supported - thus not loaded
  2. The file is supported - and still loading
  
This is especially obvious on shared drives and opening >100 MB .TIF files.

**Comment by jurplel** [comment]
Yes, might be as simple as drawing a throbber rather than trying to maintain progress, since I don't think QImageReader exposes anything.
